RestaurantReservation.Db/Services/EmployeeService.cs

```diff
@@ -0,0 +1,49 @@
+﻿using RestaurantReservation.Db.Models;
+using RestaurantReservation.Db.Repositories;
+
+namespace RestaurantReservation.Db.Services
+{
+    public class EmployeeService : IEmployeeService
+    {
+        private RestaurantReservationDbContext _context;
+        private EmployeeRepository _employeesRepository;
+
+        public EmployeeService(RestaurantReservationDbContext context)
+        {
+            _context = context;
+            _employeesRepository = new(_context);
+        }
+
+        public async Task<int> CreateAsync(EmployeeDTO newEmployee)
+        {
+            return await _employeesRepository.CreateAsync(newEmployee);
+        }
+
+        public async Task<EmployeeDTO> GetAsync(int employeeId)
+        {
+            return await _employeesRepository.GetAsync(employeeId);
+        }
+
+        public async Task<List<EmployeeDTO>> GetAllAsync()
+        {
+            return await _employeesRepository.GetAllAsync();
+        }
+
+        public async Task UpdateAsync(EmployeeDTO updatedEmployee)
+        {
+            await _employeesRepository.UpdateAsync(updatedEmployee);
+        }
+
+        public async Task DeleteAsync(int employeeId)
+        {
+            await _employeesRepository.DeleteAsync(employeeId);
+        }
+
+        public async Task<List<EmployeeDTO>> ListManagersAsync()
+        {
+            var employees = await GetAllAsync();
+            return employees.Where(employee => employee.Position == "Manager")
```

Comment:
We try to avoid to have hardcoded strings in code, if you want to change the name of it, will be hard to do that change to all fille, and reusing it is tricky. So please move this to a constant file call EmployeePositions. And update seed methods to use this new constant